pages/programs/index.tsx

```diff
@@ -1,23 +1,123 @@
-import { ProgramsList } from '../../lib/components/programs-list';
-import { Text, Container, Flex } from '@chakra-ui/react';
+import { ProgramListingProps, ProgramsList } from '../../lib/components/programs-list';
+import { Container, Flex } from '@chakra-ui/react';
 import OSLHeader from '../../lib/components/header';
 
+const programs: ProgramListingProps[] = [
+  {
+    title: 'MONTHLY WRITING GROUP',
+    description:
+      "Ready to focus on your writing? Join OutstandingLife's new monthly writing group. \
+    In this monthly writing workshop, we will discuss a short passage or \
+    poem that effectively uses a particular form or point of view. After \
+    considering how the technique enhances the piece, we will use some \
+    aspect of the text as a prompt to jumpstart our own writing. The \
+    focus will be on short, in-class writing and discussion. Suitable for \
+    writers with some experience. \n\n Bios of workshop leaders \n\n \
```

Comment:
For some reason, in Chrome and Microsoft Edge, I see
![image](https://user-images.githubusercontent.com/71473137/195649996-b7a5607b-8953-4396-b81a-0b2da19931e4.png)

which doesn't have bios of workshop leaders on new lines. Is this something that you guys encountered at all or is my computer just buggy?

Reply:
I don't have time to check this out right now but +1 to Jamie's comment here